onebusaway-android/src/main/java/org/onebusaway/android/ui/NavigationDrawerFragment.java

```diff
@@ -289,10 +283,10 @@ private void setSelectedNavDrawerItem(int itemId) {
     }
 
     @Override
```

Comment:
To stay consistent with the `Fragment` implementation, let's leave this as:

`public void onAttach(Activity activity) {`